controllers/kbsconfig_controller.go

```diff
@@ -553,6 +574,12 @@ func (r *KbsConfigReconciler) buildKbsContainer(volumeMounts []corev1.VolumeMoun
 		},
 		// Add volume mount for KBS config
 		VolumeMounts: volumeMounts,
+		Env: []corev1.EnvVar{
+			{
+				Name:  "RUST_LOG",
+				Value: "debug",
+			},
+		},
```

Comment:
This makes `debug` as the default with no way to change it dynamically since the reconciler will edit the spec. We'll have to figure out a way to make these configurable.

Reply:
Commented out for the time being